### DISCUSSION TITLE
How to enable external images?

### DISCUSSION THREAD

**Discussion by dodanex** [Baroshem/nuxt-security#158] [Q&A] [answered]
Hello @Baroshem, how do you allow external images?

I have this setting, but it's not working:

```js
security: {
  contentSecurityPolicy: {
    'img-src': ["'self'", 'https:', 'data:']
  }
}
```

Thanks you!

**Comment by Baroshem** [comment, marked as answer]
Hey @dodanex 

Take a look at this issue and my comment there. It should help you setup your images content security policy :) 

https://github.com/Baroshem/nuxt-security/issues/138#issuecomment-1497883915

If you have any additional questions, please let me know.

**Reply by dodanex** [reply]
Thank you, it's working!
